cscs-checks/microbenchmarks/cpu/simd/vc.py

```diff
@@ -77,6 +77,9 @@ def __init__(self):
             'eiger:mc': {
                 'speedup': (1.32, -0.2, 0.2, '')
             },
+            'pilatus:mc': {
+                'speedup': (1.32, -0.2, 0.2, '')
+            },
```

Comment:
Suggested change:
```
@@ -31,7 +31,11 @@ class VcSimdTest(rfm.RegressionTest):

     def __init__(self):
         self.valid_systems = ['*']
-        self.valid_prog_environs = ['builtin']
+        if self.current_system.name in ['eiger', 'pilatus']:
+            self.valid_prog_environs = ['cpeGNU']
+        else:
+            self.valid_prog_environs = ['PrgEnv-gnu']
```

Reply:
Thanks @jgphpc, could you please use the GitHub feature "Suggest change"? It make it easier to accept the changes. 
See https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request